Servers/UI/OJS.Servers.Ui/ClientApp/src/components/submissions/submission-grid-row/SubmissionGridRow.module.scss

```diff
@@ -70,34 +18,63 @@
   border-bottom: 1px solid #212328;
 }
 
-.container {
-  @extend %font-family-roboto;
+.redirectButton {
+  background-color: inherit;
+  border: 0;
+  color: colors.$light-blue-color;
+  cursor: pointer;
+  font: inherit;
+  font-size: variables.$f-size-16;
+  padding-left: 0;
+  text-align: left;
+}
+
+.link {
+  @extend %r-heading-primary;
+
+  color: colors.$light-blue-color;
+  display: inline-block;
+  margin: 0 5px;
```

Comment:
we can use spacings.$sp-value here, instead of just a number, for consistency. If the value is not present in the spacings file, using the closest one to the desired value would also be fine, that applies fot the gap prop as well